target/linux/realtek/base-files/lib/upgrade/platform.sh

```diff
@@ -35,5 +35,14 @@ platform_do_upgrade() {
 	*)
 		default_do_upgrade "$1"
 		;;
+	plasmacloud,psx8|\
+	plasmacloud,psx10)
+		PART_NAME="inactive"
+		platform_do_upgrade_dualboot_datachk "$1"
+		;;
+	*)
```

Comment:
The default case must be always at the end - this part is correct. But you've duplicated the default case and forgot to remove the one before ` plasmacloud,psx8|\`